composer.json

```diff
@@ -1,48 +1,55 @@
 {
-    "name": "contao-community-alliance/url-builder",
-    "description": "General purpose URL builder and manipulator.",
-    "license": "LGPL-3.0-or-later",
-    "authors": [
-        {
-            "name": "Christian Schiffler",
-            "email": "c.schiffler@cyberspectrum.de",
-            "homepage": "http://www.cyberspectrum.de",
-            "role": "Developer"
-        },
-        {
-            "name": "Stefan Heimes",
-            "email": "stefan_heimes@hotmail.com",
-            "role": "Developer"
-        }
-    ],
-    "support": {
-        "email": "info@contao-community-alliance.de",
-        "issues": "https://github.com/contao-community-alliance/url-builder/issues",
-        "irc": "irc://irc.freenode.org/contao.cca",
-        "source": "https://github.com/contao-community-alliance/url-builder"
+  "name": "contao-community-alliance/url-builder",
+  "description": "General purpose URL builder and manipulator.",
+  "license": "LGPL-3.0-or-later",
+  "authors": [
+    {
+      "name": "Christian Schiffler",
+      "email": "c.schiffler@cyberspectrum.de",
+      "homepage": "https://www.cyberspectrum.de",
+      "role": "Developer"
     },
-    "require": {
-        "php": "^5.6 || ^7.0"
-    },
-    "require-dev": {
-        "phpcq/all-tasks": "^1.2",
-        "contao/core-bundle": "^4.4",
-		"php-http/guzzle6-adapter": "^1.1"
-    },
-    "autoload": {
-        "psr-4": {
-            "ContaoCommunityAlliance\\UrlBuilder\\": "src/"
-        }
-    },
-    "autoload-dev": {
-        "psr-4": {
-            "ContaoCommunityAlliance\\UrlBuilder\\Test\\": "tests/"
-        }
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "1.3.x-dev",
-            "dev-develop": "1.4.x-dev"
-        }
+    {
+      "name": "Stefan Heimes",
+      "email": "stefan_heimes@hotmail.com",
+      "role": "Developer"
+    }
+  ],
+  "support": {
+    "email": "info@contao-community-alliance.de",
+    "issues": "https://github.com/contao-community-alliance/url-builder/issues",
+    "irc": "irc://irc.freenode.org/contao.cca",
+    "source": "https://github.com/contao-community-alliance/url-builder"
+  },
+  "require": {
+    "php": "^5.6 || ^7.0 || ^8.0"
+  },
+  "require-dev": {
+    "contao/core-bundle": "^4.4",
+    "php-http/guzzle6-adapter": "^1.1",
+    "phpcq/runner-bootstrap": "^1.0@dev",
+    "symfony/security-csrf": "^4.4 || ^5.2"
```

Comment:
We probably should add conflicts for `contao/core-bundle: <4.4 || >=5.0` and `symfony/security-csrf: <4.4 || >= 6.0`

Reply:
Fixed in [cedde42](https://github.com/contao-community-alliance/url-builder/pull/4/commits/cedde42e9f49727024cf567a22df3a8ec5e52efd)